### DISCUSSION TITLE
Suggestion : put a link to the tutorial on the home page's top navigation bar

### DISCUSSION THREAD

**Discussion by serge-hulne** [mint-lang/mint#688] [General]
Suggestion : put a link to the tutorial (https://tutorial.mint-lang.com) on the home page's top navigation bar (https://mint-lang.com).

**Comment by gdotdesign** [comment]
Hi :wave: 

Thank you for the suggestion! I'm in the process of redesigning the website and the tutorial will be an integral part of it. I'll close this discussion when the new website is out.
